### PR TITLE
Update hardware section

### PR DESCRIPTION
**Pull request by wled-faq**
Separate hardware into LED Strips, controllers and others, update content.
Tested with mkdocs build/xampp apache server.

### CODE REVIEW

File: docs/basics/compatible-led-strips.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: Compatible LED strips
+hide:
+  # - navigation
+  # - toc
+---
+
+WLED supports two types of LED strips: the so-called digital addressable LED strips and the so-called analog non-addressable LED strips.
+
+## Addressable LED Strips
```
Comment (blazoncek) [inline docs/basics/compatible-led-strips.md:10]: Please also add LPD6803, P9813, TM1829, UCS8903 and UCS8904

File: docs/basics/compatible-controllers.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+title: Compatible controllers
+hide:
+  # - navigation
+  # - toc
+---
+
+Currently (as of summer 2023) only the microcontrollers from Espressif, types ESP8266 and ESP32 (with dual core) are fully supported. Some of the newer types like ESP32-C3, ESP32-S2 and ESP32-S3 are experimentally supported, but productive use is not recommended yet because some bugs are suspected. In addition, the installation of these newer types is not that easy: there are several hardware variations/boards for which the SW must be specially compiled. It should also be mentioned that some ESP32-C3 boards (so-called C3 mini V1.0.0) are still being sold on the market where WLAN does not work properly.
+
+To try out WLED with 5V addressable strip a simple microcontroller board like ESP8266 D1 mini or ESP32 D1 mini is sufficient. The simplest setup might be like:
+![Simple WLED setup](https://wled-faq.github.io/pictures/pic10.jpg)
```
Comment (blazoncek) [inline docs/basics/compatible-controllers.md:11]: I would add images (and you can put your copyright on them if you wish) to the KB and not use external links if possible.

**Review by blazoncek — APPROVED** [review]
Approved if renders correctly. 😄

File: docs/basics/compatible-controllers.md (hunk 2 above)
Comment (wled-faq) [reply docs/basics/compatible-controllers.md:11]: @blazoncek added images locally

**Pull request merged**